lib/easyredmine_budget_quotas/time_entry_validation.rb

```diff
@@ -71,41 +74,45 @@ def check_if_budget_quota_valid
 
             assignable_hours = assignable_value/value_per_hour
 
-            # get current indx from comment
-            comment_id = self.comments.match(/(?<=\[)[0-9]{1,}/)
+            # get current index from comment
```

Comment:
@florianeck 
just fix a crash, but 1. this is not working, 2. this is no solution